include/superior_mysqlpp/prepared_statements/initialize_bindings.hpp

```diff
@@ -14,14 +14,23 @@
 
 namespace SuperiorMySqlpp
 {
+    /*
+     * Objective of this whole file is proper initialization of bindings.
+     * Namely, filling of MYSQL_BIND structures that tell the C MySQL client what
+     * type given parameter/result field has and where is it stored.
+     */
     namespace detail
     {
+        /**
+         * Trait specialization saying that std::string can be used as parameter
+         * for string field type.
+         */
         template<>
         struct CanBindAsParam<BindingTypes::String, std::string> : std::true_type {};
 
 
-        /*
-         * Common bindings
+        /**
```

Comment:
It would be helpful here, to have more thorough description, what is done in the function and that this is only internal part of actual initialization(because Nullable<T> must have also initialize its "content" T)